src/steamship/invocable/lambda_handler.py

```diff
@@ -27,194 +28,244 @@ def encode_exception(obj):
     return obj
 
 
-def create_handler(invocable_cls: Type[Invocable]):  # noqa: C901
-    """Wrapper function for a Steamship invocable within an AWS Lambda function."""
+def internal_handler(  # noqa: C901
+    invocable_cls_func: Callable[[], Type[Invocable]],
+    event: Dict,
+    client: Steamship,
+) -> InvocableResponse:
 
-    def _handler(
-        event: Dict,
-        client: Steamship,
-        _: Dict = None,
-    ) -> InvocableResponse:
+    try:
+        request = InvocableRequest.parse_obj(event)
+    except SteamshipError as se:
+        logging.exception(se)
+        return InvocableResponse.from_obj(se)
+    except Exception as ex:
+        logging.exception(ex)
+        return InvocableResponse.error(
+            code=HTTPStatus.INTERNAL_SERVER_ERROR,
+            message="Plugin/App handler was unable to parse inbound request.",
+            exception=ex,
+        )
 
-        try:
-            request = InvocableRequest.parse_obj(event)
-        except SteamshipError as se:
-            logging.exception(se)
-            return InvocableResponse.from_obj(se)
-        except Exception as ex:
-            logging.exception(ex)
-            return InvocableResponse.error(
-                code=HTTPStatus.INTERNAL_SERVER_ERROR,
-                message="Plugin/App handler was unable to parse inbound request.",
-                exception=ex,
-            )
+    if request and request.invocation:
+        error_prefix = (
+            f"[ERROR - {request.invocation.http_verb} {request.invocation.invocation_path}] "
+        )
+    else:
+        error_prefix = "[ERROR - ?VERB ?PATH] "
 
-        if request and request.invocation:
-            error_prefix = (
-                f"[ERROR - {request.invocation.http_verb} {request.invocation.invocation_path}] "
-            )
-        else:
-            error_prefix = "[ERROR - ?VERB ?PATH] "
-
-        try:
-            invocable = invocable_cls(client=client, config=request.invocation.config)
-        except SteamshipError as se:
-            return InvocableResponse.from_obj(se)
-        except Exception as ex:
-            logging.exception(ex)
-            return InvocableResponse.error(
-                code=HTTPStatus.INTERNAL_SERVER_ERROR,
-                prefix=error_prefix,
-                message="Unable to initialize package/plugin.",
-                exception=ex,
-            )
+    try:
+        invocable = invocable_cls_func()(client=client, config=request.invocation.config)
+    except SteamshipError as se:
+        return InvocableResponse.from_obj(se)
```

Comment:
Do we want to log this exception too? 

```suggestion
        logging.exception(ex)
        return InvocableResponse.from_obj(se)
```

Reply:
Yup.  Added it to next PR.